shared/src/commonMain/kotlin/io.newm.shared/internal/implementations/LoginUseCaseImpl.kt

```diff
@@ -47,7 +47,7 @@ internal class LoginUseCaseImpl(
         }
     }
 
-    override fun logout() {
+    override suspend fun logout() {
         connectWalletUseCase.disconnect()
```

Comment:
We won't want to disconnect the wallet anymore right?

Reply:
Not anymore, since the wallet is connected we will keep it that way so next time they login there is a wallet associated with the account.